src/themes/components/index.ts

```diff
@@ -15,6 +15,7 @@ import Input from './input';
 import FormLabel from './formLabel';
 import FormError from './formError';
 import Form from './form';
+import Drawer from './Drawer';
```

Comment:
Created capitalized filename, because of eslint:

```
error  Filename is not in pascal case. Rename it to `Drawer.tsx`  unicorn/filename-cas
```

Other files in this folder are lower case, not sure if we want to skip this rule

Reply:
It's strange that it doesn't complain for the other files.
Would you mind to take another look here?

Thanks!

Reply:
@GutierrezGo fixed that